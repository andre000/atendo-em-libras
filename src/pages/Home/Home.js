import React from 'react'
import { Container } from '../../components/Container'
import { ProviderList } from '../../components/ProviderList'
import { HomeCallOut } from './HomeCallOut/HomeCallOut'

class Home extends React.Component {
  render() {
    return (
      <>
        <HomeCallOut />
        <Container>
          <ProviderList />
        </Container>
      </>
    )
  }
}

export { Home }
