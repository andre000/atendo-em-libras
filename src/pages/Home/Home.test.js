import React from 'react'
import { Home } from './Home'
import { render } from '@testing-library/react'
import 'jest-styled-components'

describe('Home component', () => {
  let view

  beforeEach(() => {
    view = render(<Home />)
  })

  it('renders home correctly', () => {
    expect(view).toMatchSnapshot()
  })
})
