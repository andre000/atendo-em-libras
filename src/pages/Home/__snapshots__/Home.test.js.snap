// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders home correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": .c1 {
  max-width: 1500px;
  width: 100%;
  box-sizing: border-box;
  padding: 0 80px;
  margin: 0 auto;
}

.c0 {
  width: 100%;
  min-height: 220px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: column;
  -ms-flex-flow: column;
  flex-flow: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  box-sizing: content-box;
  background-color: #ffffff;
}

.c0:before {
  content: '';
  height: 8px;
  width: 65px;
  margin-bottom: 32px;
  background-color: #5996f7;
  border-radius: 4px;
}

.c0 h2 {
  max-width: 80%;
  color: #000000;
  text-align: center;
  font-size: 2rem;
  font-weight: 400;
  line-height: 2.8rem;
}

@media (max-width:800px) {
  .c1 {
    padding: 0 10vw;
  }
}

@media (min-width:800px) {
  .c0 h2 {
    max-width: 50%;
    font-size: 3.2em;
    line-height: 5rem;
  }
}

<body>
    <div>
      <div
        class="c0"
      >
        <h2>
          Profissionais de diversas áreas da saúde prontos para te atender
        </h2>
      </div>
      <div
        class="c1"
      >
        <div
          data-testid="emptyList"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="HomeCallOut__StyledHomeCallOut-sc-8y0p5g-0 inrJok"
    >
      <h2>
        Profissionais de diversas áreas da saúde prontos para te atender
      </h2>
    </div>
    <div
      class="Container-kkckgz-0 fUhoJX"
    >
      <div
        data-testid="emptyList"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
